tests/php/Shortcodes/FileShortcodeProviderTest.php

```diff
@@ -179,4 +181,30 @@ public function testMarkupHasStringValue()
             'Test that shortcode with invalid file ID is not parsed.'
         );
     }
+
+    public function testCacheHit()
+    {
+        $testFile = $this->objFromFixture(File::class, 'asdf');
+        $testFileID = $testFile->ID;
+        $tuple = $testFile->File->getValue();
+
+        // Prewarm cache with non-sense value
+        $markup = '<img src="cacheHit">';
```

Comment:
This is a mildly dumb way of doing this, but short of re-factoring the shortcode handler to use dependency injection, it will have to do.